### PR TITLE
Add MathExp-kpetrov2

### PR DESCRIPTION
**Pull request by kpetrov2**
### Description

I have created a directory named exp/ in the section specified by the issue advertiser. Inside this directory is the exp.md file that contains the necessary information to understand the Math.Exp() method.
I have chosen this issue to contribute for the first time to the CodeCademy Docs in order to finish the GitHub course. I appreciate the chance to participate in a real task.

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+Title: '.Exp()'
+Description: 'Returns the result of raising e with a specific number.'
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/exp/exp.md:3]:
```suggestion
Description: 'Returns the result of raising e to the power of a specific number.'
```

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Exp()`** method returns the value of raising **`e`** with the number given as a **power**.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/exp/exp.md:16]:
```suggestion
The **`Math.Exp()`** method returns the value of raising _e_ to the power of a given number.
```

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+**`e`** stands for Euler's number, the **Euler's number** is an irrational number, with an approximate constant value of **2.71828** , that provides a constant base to the **natural logarithm** or **`log`** and can be understood as a limit of a given progression.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/exp/exp.md:17]:
```suggestion
_e_ stands for Euler's number, an irrational number, with an approximate constant value of 2.71828. It provides a constant base to the natural logarithm or log and can be understood as a limit of a given progression.
```

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+$$
+\sum\limits_{n=0}^{\infty} \frac{1}{n!} = 1 + \frac{1}{n+1} + \frac{1}{(n+1)(n+2)} + \frac{1}{(n+1)(n+2)(n+3)} + \cdots
+$$
+
+Consequently, the **`Math.Exp()`** method is the inverse of the **`Math.Log()`** method.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/exp/exp.md:23]:
```suggestion
Consequently, the `Math.Exp()` method is the inverse of the `Math.Log()` method.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kpetrov2, I did a first review and left a few comments.

**Comment by kpetrov2** [comment]
I've made the changes, should I wait or should I do another pull request?

**Comment by yangc95** [comment]
> I've made the changes, should I wait or should I do another pull request?

@kpetrov2 Please see: https://github.com/Codecademy/docs/issues/1606#issuecomment-1430853336

File: content/c-sharp/concepts/math-functions/terms/exp/exp.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Exp()`** method returns the value of raising _e_ the power of a given number.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/exp/exp.md:16]:
```suggestion
The **`Math.Exp()`** method returns the value of raising _e_ to the power of a given number.
```

**Review by SSwiniarski — APPROVED** [review]
@kpetrov2 One minor typo, otherwise it looks good for a second review.

**Comment by kpetrov2** [comment]
Correct! Sorry I forgot that, thanks for the advice :)

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@kpetrov2 here is the live link for this one: https://www.codecademy.com/resources/docs/c-sharp/math-functions/exp

Would you be able to make an additional PR to resolve the issue with this section: `$$ \sum\limits_{n=0}^{\infty} \frac{1}{n!} = 1 + \frac{1}{n+1} + \frac{1}{(n+1)(n+2)} + \frac{1}{(n+1)(n+2)(n+3)} + \cdots $$`

Maybe an image can be swapped out for that line instead!

You're also currently anonymous:
Screen Shot 2023-02-18 at 12 21 39 AM
Make sure to add your GitHub username to your Codecademy profile and set the visibility to "everyone."

cc: @SSwiniarski @KTom101

**Pull request merged**